cves/kernel/CVE-2021-43267.yml

```diff
@@ -235,10 +243,10 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+    - commit:
```

Comment:
If there is no interesting commits, then there should be an explanation of why. Rudimentary fixing? Single commit fix?